docs/reference/adobe/experience/decisioning/scope-details.schema.json

```diff
@@ -55,6 +55,16 @@
                     "title": "Selected Experience Option",
                     "description": "The experience that was selected when the decision was made.",
                     "$ref": "#/definitions/experience-option-details"
+                },
+                "xdm:characteristics": {
```

Comment:
DO you have list of specific keys that you want to populate in this map type ?

Reply:
currently we are thinking of `tnta` and `state` we should still decide on naming, but we would like to use this object to send various details related to Target, ODS and future AJO